__tests__/app.test.js

```diff
@@ -465,6 +465,119 @@ describe('POST /api/articles/:article_id/comments', () => {
     });
 });
 
+describe('PATCH /api/articles/:article_id', () => {
+    test('200: should have updated article with correct object layout', () => {
+        const objectLayout = {
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String)
+        };
+
+        return request(app)
+        .patch('/api/articles/1')
+        .send({ inc_votes: 1 })
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.article).toMatchObject(objectLayout);
+        });
+    });
+
+    test('number of votes should increase by specified amount', () => {
```

Comment:
You've also decreased in this test - so maybe that part should be a separate test?